affordance_primitives/include/affordance_primitives/screw_planning/screw_planning.hpp

```diff
@@ -55,12 +55,19 @@ bool constraintFn(
 
 Eigen::VectorXd calcError(const Eigen::Isometry3d & tf_err);
 
+//double calcErrorDerivative(
+//  const Eigen::Isometry3d & tf_current_to_start, const double current_theta,
+//const ScrewAxis & screw_axis);
+
 double calcErrorDerivative(
-  const Eigen::Isometry3d & tf_current_to_start, const double current_theta,
+  const Eigen::Isometry3d & tf_m_to_q, const Eigen::Isometry3d & tf_m_to_e, const double current_theta,
```

Comment:
The paper uses "m" for "map" frame... I wonder if we should update the names here to make it more general. I think of the planning frame, but we already have "p" meaning "path" so I don't love that. What do you think?